linerate/models/thermal_model.py

```diff
@@ -262,12 +262,12 @@ def compute_conductor_temperature(
         Union[float, float64, ndarray[Any, dtype[float64]]]
             :math:`I~\left[\text{A}\right]`. The thermal rating.
         """
+        n = self.span.num_conductors
         T = solver.compute_conductor_temperature(
             self.compute_heat_balance,
-            current=current,
+            current=current / n,
```

Comment:
Kanskje vi skal spesifisere i docstringen lenger opp at current er total current for alle ledere?